### ISSUE TITLE
[Accesibility] Empty button issue with dropdown icon in Categories List

### ISSUE THREAD

**Issue by kivuvarosekivuvan** [ushahidi/platform#4979]
# Engineers Checklist

## Describe the bug
- The dropdown icon button in the list of /settings/categories route is empty, which means that it does not have any descriptive text. This can be an accessibility issue, as screen reader users may not be able to understand the purpose of the button.

## To Reproduce

-  Navigate to the page where the mzima-client-button component is used : http://localhost:4200/settings/categories
-  Inspect the button with the category-item--expand class.
-  Observe that the button is empty and does not have any text content.
-  Alternatively, use a accessibility tool to test like WAVE or use screen reader to read out the purpose of the button

## Expected behavior
The button should have a descriptive text content or an appropriate value attribute to indicate its purpose to screen reader users.

## How to solve

1. Add a text label inside the <mzima-client-button> element, or set the value attribute of the button.
2. Alternatively, you can use the aria-label or aria-labelledby attributes to provide a descriptive label for the button.

## Screenshots
<img width="1595" alt="Screenshot 2024-07-23 at 08 51 28" src="https://github.com/user-attachments/assets/2ec09782-d339-4c24-bc7f-e05ac21ab295">
<img width="1595" alt="Screenshot 2024-07-23 at 08 52 04" src="https://github.com/user-attachments/assets/207d5dad-c4e3-48e6-aa3e-1d9587766e3c">
